reactDashboard/src/components/Utils/CsvUtils.test.js

```diff
@@ -427,24 +402,33 @@ const dailyCasesCsvData = `
     `;
 
 describe("getNhsCsvDataDateRange", () => {
-
   const parsedDailyHealthBoardData = readCsvData(dailyHealthBoardCsvData);
-  const parsedDailyCouncilAreaData = readCsvData(dailyCouncilAreaCsvData);
-  const expectedResult = {startDate: Date.parse("2020-03-06"), endDate: Date.parse("2020-03-09")}
-
-  it("happy path", () => {
-    expect(getNhsCsvDataDateRange(parsedDailyHealthBoardData, parsedDailyCouncilAreaData)).toStrictEqual(
-      expectedResult
-    );
-  })
+  const parsedDailyCouncilAreaData = readCsvData(
+    dailyCouncilAreaCsvData +
+      `
+  20200408,S08000031,place 1,0,27,0.27,0,7,70,0,37,0
```

Comment:
what is this for?

Reply:
The test didn't actually check that the date range encompassed both csv inputs. We could have ignored the CA csv file in the code and the test would have still passed. This addition ensures that both files would have had to have been used

Reply:
Thanks :)